batch/sc-batch-fargate.yaml

```diff
@@ -60,11 +60,8 @@ Parameters:
   Command:
     Type: String
     Description: >
-      A pipe separated list which is the command to pass to the container (i.e. echo|hello|world)
-    ConstraintDescription: >
-      Must be a pipe delimited list of strings
-    Default: "echo|hello|world"
-    AllowedPattern: "^(?!s*$).+"
+      (Optional) The command that's passed to the docker container (i.e. echo hello world).
```

Comment:
Grammar:  Should be "e.g., echo hello world", not "i.e. echo hello world".

Reply:
fixed